app/partials/main-footer.cjsx

```diff
@@ -2,15 +2,55 @@ counterpart = require 'counterpart'
 React = require 'react'
 Translate = require 'react-translate-component'
 {Link} = require 'react-router'
+ZooniverseLogo = require './zooniverse-logo'
 
 counterpart.registerTranslations 'en',
   footer:
-    privacyPolicy: 'Privacy policy'
+    discover:
+      title: 'Discover'
+      projectList: 'Project List'
+    learn:
+      title: 'Learn'
+      aboutUs: 'About Us'
+      ourTeam: 'Our Team'
+      education: 'Education'
+      privacyPolicy: 'Privacy policy'
+    discuss:
+      title: 'Discuss'
+      zooTalk: 'Zooniverse Talk'
+      otherTalk: 'Other Talk'
 
 module.exports = React.createClass
   displayName: 'MainFooter'
 
   render: ->
     <footer className="main-footer">
-      <Link to="privacy"><Translate content="footer.privacyPolicy" /></Link>
+      <div className="centered-grid main-footer-flex">
+        <Link to="home" className="main-logo">
+          Zo<ZooniverseLogo />niverse
```

Comment:
This logo layers a few shapes on top of each other, so when you set `color` to something translucent, the overlap shows:

![screen shot 2015-05-14 at 4 12 38 pm](https://cloud.githubusercontent.com/assets/464389/7642118/1450c992-fa54-11e4-8cc7-ee8cf30e431c.png)

I'd either use a solid text color and change the opacity of its container or redraw the logo using a single shape.